client/state/support/selectors.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @param  {Object} state Global state tree
+ * @return {Object}       The support section of the state
+ */
+export function getSupportState( state ) {
+	return state.support;
+}
```

Comment:
Oops, I recently switched to Sublime which doesn't add them by default :) Fixed.

Also just found this Sublime config option:
`"ensure_newline_at_eof_on_save": true`